src/components/elements/payment-methods.js

```diff
@@ -1,38 +1,65 @@
 import React from 'react'
 import styled from 'styled-components'
-// Icons
-import Visa from 'images/svg/visa.svg'
-import Master from 'images/svg/mastercard.svg'
-import Neteller from 'images/svg/neteller.svg'
-import Skrill from 'images/svg/skrill.svg'
-import Maestro from 'images/svg/maestro.svg'
-import BankTransfer from 'images/svg/bank-transfer.svg'
-import InternetBank from 'images/svg/internet-bank-transfer.svg'
+import AutoCarousel from './auto-carousel'
+import { SectionContainer } from 'components/containers/container'
+import WebMoney from 'images/svg/payment-webmoney.svg'
+import Sticpay from 'images/svg/payment-sticpay.svg'
+import Banktransfer from 'images/svg/payment-bank-transfer.svg'
+import Qiwi from 'images/svg/payment-qiwi.svg'
+import Yandex from 'images/svg/payment-yandex.svg'
+import Mastercard from 'images/svg/payment-mastercard.svg'
+import Skrill from 'images/svg/payment-skrill.svg'
+import PerfectMoney from 'images/svg/payment-perfect-money.svg'
+import Fasapay from 'images/svg/payment-fasapay.svg'
+import Payscale from 'images/svg/payment-payscale.svg'
+import Netller from 'images/svg/payment-neteller.svg'
+import Maestro from 'images/svg/payment-maestro.svg'
+import InternetBankTransfer from 'images/svg/payment-internet-bank-transfer.svg'
+import Visa from 'images/svg/payment-visa.svg'
+import IWallet from 'images/svg/payment-i-wallet.svg'
+import Paytrust from 'images/svg/payment-paytrust.svg'
+import Jeton from 'images/svg/payment-jeton.svg'
 
-const Grid = styled.article`
-    padding: 2rem 0;
-    display: grid;
-    grid-template-columns: repeat(7, 1fr);
-    grid-gap: 2rem;
-    width: 100%;
-
-    * {
-        max-width: 100%;
-        justify-self: center;
-        align-self: center;
-    }
+const PaymentSection = styled(SectionContainer)`
+    padding: 2.4rem 0;
 `
-
-const PaymentMethods = () => (
-    <Grid>
-        <BankTransfer />
-        <InternetBank />
-        <Neteller />
-        <Master />
-        <Visa />
-        <Maestro />
-        <Skrill />
-    </Grid>
-)
-
+class PaymentMethods extends React.Component {
+    state = {
```

Comment:
why does this component need state?

Reply:
Because we need to change the state after every transition, after changing the state will update and our props (which is feed by the state) changes and our carousel start changing

Reply:
i think we can pass in all those logo components as the children, and we can handle it on the moving carousel state instead

Reply:
as per discussion, handled in ticker PR